owtf/plugin/plugin_handler.py

```diff
@@ -552,7 +552,7 @@ def process_plugins_for_target_list(self, plugin_group, status, target_list):
                                 {'SomeAborted': False, 'SomeSuccessful': False, 'AllSkipped': True},
                                 {target})
         else:
-            pass
+            exit(0)
```

Comment:
I am not sure about the context. Is the user expecting OWTF to exit in such case? Is it an actual success?

Reply:
Not needed, was just for debugging. Removing this.